modules/bitgo/src/v2/coins/hbar.ts

```diff
@@ -221,9 +222,48 @@ export class Hbar extends BaseCoin {
     const tx = await txBuilder.build();
     const txJson = tx.toJson();
 
-    if ((tx as any)._txBody.data !== 'cryptoTransfer') {
-      // don't explain this
-      throw new Error('Transaction format outside of cryptoTransfer not supported for explanation.');
+    let outputAmount = new BigNumber(0);
+    const outputs: { address: string; amount: string; memo: string; tokenName?: string }[] = [];
+    // TODO(BG-24809): get the memo from the toJson
+    let memo = '';
+    if (params.memo) {
+      memo = params.memo.value;
+    }
+
+    switch (txJson.instructionsData.type) {
+      case 'cryptoTransfer':
+        const recipients = txJson.instructionsData.params.recipients || [];
+        recipients.map((recipient) => {
```

Comment:
did you mean to use `forEach` here? `map` should return something. Can we clean this up and either use 
```
outputs = recipients.map(...)
```
or 
````
recipients.forEach(...)